.circleci/config.yml

```diff
@@ -109,18 +109,22 @@ jobs:
           working_directory: /home/circleci
           command: |
             echo "Master theme branch is ${CIRCLE_BRANCH}"
-            MASTER_THEME_BRANCH=dev-${CIRCLE_BRANCH} MERGE_SOURCE=git@github.com:greenpeace/planet4-base-fork.git MERGE_REF=develop make
+            MASTER_THEME_BRANCH=dev-${CIRCLE_BRANCH} MERGE_SOURCE=git@github.com:greenpeace/planet4-base-fork.git MERGE_REF=planet-5275 make
       - run:
           name: Test - Clone planet4-docker-compose
           command: |
-            git clone https://github.com/greenpeace/planet4-docker-compose
+            git clone --single-branch --branch planet-5275 https://github.com/greenpeace/planet4-docker-compose
       - run:
           name: Test - Run tests
           command: |
             export BUILD_TAG="build-$(cat /tmp/workspace/var/circle-build-num)"
             export APP_IMAGE=gcr.io/planet-4-151612/planet4-base-app:${BUILD_TAG}
             export OPENRESTY_IMAGE=gcr.io/planet-4-151612/planet4-base-openresty:${BUILD_TAG}
             make -C planet4-docker-compose ci test
+      - run:
+          name: Test - Run accessibility report
+          command: |
+            make -C planet4-docker-compose test-pa11y
```

Comment:
Do you think, this can be broken in into a separate job? (eg. a11y-tests). That way it can run in parallel, but most importantly it would allow us to merge this and have the tests run without blocking PRs until we fix all the issues reported.

Reply:
So I've made it run in another job, and [it seems to work](https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/1606/workflows/f19ad42a-2edb-44fc-b852-ae4797c260a5), but because I still want to have a report of warnings (and maybe notices), I'm doing something not pretty with jq and an exit code, don't know if there's a better way.  
What I'm wondering is, how can I configure this job so that it fails without blocking a PR merge ?

Reply:
I just adjusted repo settings, so instead of requiring the whole `main` workflow to pass it would require only the 3 jobs we want. We will still see a11y job check, but it won't block the merge button.